public/index.html

```diff
@@ -0,0 +1,56 @@
+<!-- <!DOCTYPE html>
```

Comment:
Delete unused or commented code

Reply:
ma'am, Index.html as well as readme is updated as directed , kindly review for any further changes,